packages/test/test-service-load/src/utils.ts

```diff
@@ -40,40 +40,48 @@ import { ILoadTestConfig, ITestConfig } from "./testConfigFile";
 const packageName = `${pkgName}@${pkgVersion}`;
 
 class FileLogger implements ITelemetryBufferedLogger {
-	private static readonly loggerP = new LazyPromise<FileLogger>(async () => {
-		if (process.env.FLUID_TEST_LOGGER_PKG_PATH !== undefined) {
-			await import(process.env.FLUID_TEST_LOGGER_PKG_PATH);
-			const logger = getTestLogger?.();
-			assert(logger !== undefined, "Expected getTestLogger to return something");
-			return new FileLogger(logger);
-		} else {
-			return new FileLogger();
-		}
-	});
+	private static readonly loggerP = (minLogLevel?: LogLevel) =>
+		new LazyPromise<FileLogger>(async () => {
+			if (process.env.FLUID_TEST_LOGGER_PKG_PATH !== undefined) {
+				await import(process.env.FLUID_TEST_LOGGER_PKG_PATH);
+				const logger = getTestLogger?.();
+				assert(logger !== undefined, "Expected getTestLogger to return something");
+				return new FileLogger(logger, minLogLevel);
+			} else {
+				return new FileLogger(undefined, minLogLevel);
+			}
+		});
 
-	public static async createLogger(dimensions: {
-		driverType: string;
-		driverEndpointName: string | undefined;
-		profile: string;
-		runId: number | undefined;
-	}) {
+	public static async createLogger(
+		dimensions: {
+			driverType: string;
+			driverEndpointName: string | undefined;
+			profile: string;
+			runId: number | undefined;
+		},
+		minLogLevel: LogLevel = LogLevel.default,
+	) {
+		const logger = await this.loggerP(minLogLevel);
 		return createChildLogger({
-			logger: await this.loggerP,
+			logger,
 			properties: {
 				all: dimensions,
 			},
 		});
 	}
 
 	public static async flushLogger(runInfo?: { url: string; runId?: number }) {
-		await (await this.loggerP).flush(runInfo);
+		await (await this.loggerP()).flush(runInfo);
```

Comment:
Is this actually necessary? Looks like the return of `await this.loggerP` is an instance of `FileLogger` which shouldn't need to be called as a function before calling flush on it.

Reply:
Actually, I needed to make it a function because the other place where it is called needs to supply the log level to the FileLogger.

Reply:
Totally right, I misread it.